vmdb/lib/extensions/ar_to_model_hash.rb

```diff
@@ -101,6 +101,12 @@ def to_model_hash_recursive(options, result = nil)
     return result
   end
 
+  def iterable?(klass, association)
+    reflection = klass.virtual_reflection(association) ||
+      klass.reflect_on_association(association)
```

Comment:
There is a method in the ar_virtual code called `reflections_with_virtual`.  The idea of this method was to be used in places where you are not sure if it's virtual or regular, so just go to this one method to get it.  I think that's what you want here.  That code uses `.reflections`, but this code uses `.reflect_on_association`.  Not sure if that matters.